region/crateria/east/West Ocean.json

```diff
@@ -11,15 +11,15 @@
   "nodes": [
     {
       "id": 1,
-      "name": "West Ocean Left Door (to Moat)",
+      "name": "Left Door",
       "nodeType": "door",
       "nodeSubType": "blue",
       "nodeAddress": "0x00189ca",
       "doorEnvironments": [{"physics": "air"}]
     },
     {
       "id": 2,
-      "name": "West Ocean Top Right Door (to Attic)",
+      "name": "Upper Right Section Top Right Door",
```

Comment:
Green Brin Main Shaft does sections like
`Top Right Door (Upper Right Section)`
and that is confusing at first read, but I think this should copy the format to be consistent.  And I like that way of writing it more.

Reply:
I was moving more towards this method, because I think it will be nicer to read. For `Bottom Right Door (Upper Right Section)`, if you read it quickly you might think it's the `Bottom Right Door`. Maybe `Upper Right Section - Bottom Right Door` would work the best?

Reply:
Right, `Bottom Left Door (Above Power Bomb Blocks)` is hard to understand at first, like here.  
`Above Power Bomb Blocks - Bottom Left Door` works.  Its just different from the majority of node names since only two or three rooms have sections.